.docker/ci/Dockerfile

```
@@ -1,50 +1,90 @@
-# moveit/moveit:melodic-ci
+# moveit/moveit2:crystal-ci
 # Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
-MAINTAINER Dave Coleman dave@picknik.ai
+LABEL maintainer="mike@picknik.ai"
 
 ENV TERM xterm
 
 # Setup catkin workspace
-ENV CATKIN_WS=/root/ws_moveit
-WORKDIR $CATKIN_WS
+ENV ROS_WS=/opt/ws_moveit
```

I'm not sure why you changed the original location at all. But /opt seems to be rather weird for a temporary and private folder. I suggest to use `ros_ws` as folder name (as everywhere else now) and `/tmp` to indicate temporary usage.
```suggestion
ENV ROS_WS=/tmp/ros_ws
```

---

Developers often mount /tmp in the container to an external volume to cache filesystems, proxies, and workspaces between the host. I don't think it would helpful to place the workspace that the Dockerfile is to build at the only root folder that is frequently overloaded from docker volumes.